src/components/CreateCourseRunPage/index.jsx

```diff
@@ -83,6 +86,26 @@ class CreateCourseRunPage extends React.Component {
       ({ label: choice.display_name, value: choice.value }));
   }
 
+  compareDates(a, b) {
+    let c;
+    if (moment(a.start).isBefore(b.start)) {
+      c = 1;
+    } else if (moment(b.start).isBefore(a.start)) {
+      c = -1;
+    } else {
+      c = 0;
+    }
+    return c;
+  }
+
+  parseCourseRunLabels(courseRuns) {
+    const sortedCourseRuns = courseRuns.sort(this.compareDates);
+    return sortedCourseRuns.map((run) => {
+      const runKey = (run.key).split('+', 3)[2];
```

Comment:
This is operating on the assumption that our course key will always be in this format: `course-v1:edX+cs101+2T2019`
Is this a safe assumption, or is there another way I can grab the course run's ID?

Reply:
It is not unfortunately. Old course keys used to be `Org/Number/Term`. I think you might need to write a regex or something to parse this. I'm not sure off the top of my head. I would maybe poke around discovery since I know we do some parsing there and it should have a way of doing it. Maybe in the course runs endpoint.

Reply:
I would also rename this variable to be `runTerm`. The thing that you are parsing is already the runKey.